### PR TITLE
Mege in work from elasticsearch-dump-next

### PR DESCRIPTION
**Pull request by evantahler**
Hello @ferronrsmith!

I see you've done a lot of good work updating this tool so it works for ElasticSearch v6, and released it under `elasticdump-next` on NPM.  Thanks!

Would you be interested in becoming a maintainer of this package and merging the projects?  [I'm looking for more folks to help manage this project](https://github.com/taskrabbit/elasticsearch-dump/issues/333).  I also think that having one package would be simpler for folks trying to use the tool, get help, and provide patches.

### CODE REVIEW

**Comment by evantahler** [comment]
(I don't know if mentioning someone in a PR description notifies them, so I'm reaching out to @ferronrsmith again here!)

**Comment by ferronrsmith** [comment]
Hey sorry. Gmail pushed all my GitHub notifications to Junk. Yea sure that would be awesome ! Let know what I need to know.

**Comment by evantahler** [comment]
Awesome! Step 1 is making you a maintainer of this project (which I just did!).  

Please compare your work to https://github.com/taskrabbit/elasticsearch-dump/pull/364, which I just merged into master.  Getting the test suite to pass for all the ElasticSearch versions seems like the first priority to get working.   Your work is similar in a lot of ways, so I merged in the oldest PR, as these 2 conflict.  

Once you have a decision on how to move forward with the tests, I'll grant you NPM publish access as well. 

CC @ebuildy and @datashaman

File: test/parentchild.js [hunk 1]
```diff
@@ -1,362 +0,0 @@
-'use strict'
```
Comment (evantahler) [inline test/parentchild.js]: Removing this test doesn't quite seem right.  Parent-child relationships are still a thing... they just can't be in the same index any more?

File: package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (evantahler) [inline package.json]: Please don't merge in these changes to `package.json`

File: lib/help.txt [hunk 3]
```diff
@@ -136,4 +136,4 @@ elasticdump \
   --searchBody '{"query":{"term":{"username": "admin"}}}'
 
 ------------------------------------------------------------------------------
-Learn more @ https://github.com/taskrabbit/elasticsearch-dump
+Learn more @ https://github.com/node-packages/elasticsearch-dump
```
Comment (evantahler) [inline lib/help.txt]: Please don't merge in these changes to `help.txt`

File: README.md [hunk 4]
```diff
@@ -1,38 +1,31 @@
-elasticdump
+elasticdump-next
```
Comment (evantahler) [inline README.md]: Please don't merge in these changes to `readme`

File: Dockerfile [hunk 5]
```diff
@@ -1,8 +1,6 @@
-FROM alpine:3.6
-LABEL maintainer="evan@evantahler.com"
+FROM node:8.9-alpine
+LABEL maintainer="ferronrsmith@gmail.com"
```
Comment (evantahler) [inline Dockerfile]: Please don't change the maintainer

File: Dockerfile [hunk 6]
```diff
@@ -1,8 +1,6 @@
-FROM alpine:3.6
-LABEL maintainer="evan@evantahler.com"
+FROM node:8.9-alpine
```
Comment (evantahler) [inline Dockerfile]: Upgrades are good!

File: test/parentchild.js (hunk 1 above)
Comment (ferronrsmith) [reply test/parentchild.js]: ES 6 dropped support for parent-child (notice the tests failing). [parent-join](https://www.elastic.co/guide/en/elasticsearch/reference/6.0/parent-join.html) has to be used now.

**Comment by ferronrsmith** [comment]
I think it's best if this issue is closed and I add the changes for ES 6 on top of a new working fork.

**Comment by ferronrsmith** [comment]
@evantahler can you resend the invite thanks

**Comment by evantahler** [comment]
check your email!

**Comment by ferronrsmith** [comment]
thanks

**Comment by ferronrsmith** [comment]
@evantahler I don't have permissions to create releases. Is that something that can be sorted. Thanks

**Comment by evantahler** [comment]
Sure is! What’s your NPM name? And please ensure that you have 2fa enabled on your NPM account.

**Comment by ferronrsmith** [comment]
ferronrsmith. yes 2fa is enabled

**Comment by evantahler** [comment]
You have been added!
screen shot 2018-01-21 at 11 26 15 am

please take down the elasticdump-next package, and direct people here!

**Pull request closed without merging**